301-eventhub-namespace-ipfilter/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "eventhubNamespaceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "ipFilterRuleName": {
+            "value": "GEN-UNIQUE"
+        },
+        "ipFilterAction": {
```

Comment:
remove this and use a defaultValue in the template

Reply:
resolved